pkg/agent/cluster.go

```diff
@@ -111,6 +112,8 @@ func NewCluster(ctx context.Context, assetDir string) (*Cluster, error) {
 	czero.assetDir = assetDir
 	czero.clusterConsoleRouteURL = ""
 	czero.installHistory = cinstallstatushistory
+	czero.installHistory.ValidationResults.ClusterValidationHistory = make(map[string]*validationResultHistory)
```

Comment:
Tests are failing with a nil pointer here, presumably because ValidationResults is not initialised.

Reply:
Yep good catch